NEWS.md

```diff
@@ -0,0 +1,55 @@
+
+# MMAction2 Release Notes
+
+## v0.6.0 (Master)
+
+### Models
+* Add NonLocal module in TSM and I3D (https://github.com/open-mmlab/mmaction2/pull/41)
+* Add SSN pipeline (https://github.com/open-mmlab/mmaction2/pull/33)
+* Add SSN losses (https://github.com/open-mmlab/mmaction2/pull/52)
```

Comment:
We can have a single item when SSN is completed. These parts are not useful for users.